File: examples/js/weather/now/getCurrentWeather.js
const { WeatherService } = require("m3o/weather");

// Get the current weather report for a location by postcode, city, zip code, ip address
async function getCurrentWeather() {
  let weatherService = new WeatherService(process.env.MICRO_API_TOKEN);
  let rsp = await weatherService.now({
    location: "london",
  });
  console.log(rsp);
}

getCurrentWeather();

File: examples/go/weather/now/getCurrentWeather.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/weather"
)

// Get the current weather report for a location by postcode, city, zip code, ip address
func GetCurrentWeather() {
	weatherService := weather.NewWeatherService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := weatherService.Now(&weather.NowRequest{
		Location: "london",
	})
	fmt.Println(rsp, err)
}
